security/maltrail/src/opnsense/mvc/app/controllers/OPNsense/Maltrail/forms/general.xml

```diff
@@ -0,0 +1,30 @@
+<form>
+    <field>
+        <id>general.heuristics</id>
+        <label>Use Heuristics</label>
+        <type>checkbox</type>
+        <help>Whether to enable or disable the usage or heuristic detection.</help>
+    </field>
+    <field>
+        <id>general.updateperiod</id>
+        <label>Update Period</label>
+        <type>text</type>
+        <help>Time in seconds how often to refresh trails.</help>
+    </field>
+    <field>
+        <id>general.adminpassword</id>
+        <label>SHA256 Admin Password</label>
+        <type>text</type>
+        <help><![CDATA[Set the SHA256 password for user admin here.<br/>
```

Comment:
With the whitespace issues and HTML tags this will never be issued to the translation. Better to put these into the documentation and try to make a brief statement here.

Reply:
I'll remove it and write a documentation page before stable release

Reply:
👍